src/apps/routes/schemas/webhooks.clj

```diff
@@ -3,11 +3,15 @@
         [schema.core :only [defschema optional-key]])
   (:import [java.util UUID]))
 
-(def WebhookIdParam (describe UUID "A UUID that is used to identify the Webhook"))
+(defschema WebhookType
+  {:id       (describe UUID "A UUID for the type")
+   :type     (describe NonBlankString "Webhook type")
+   :template (describe NonBlankString "Template for this Webhook type")})
```

Comment:
This makes the `template` required in both the webhook list responses and requests.

Are we allowing the client to update the `template` for webhook types?
If not, we can make this field optional here so that it's not required in requests.

See also my question in cyverse-de/ui#242